Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -1453,7 +1540,22 @@ struct molops_wrapper {
          python::arg("branchedPaths") = true,
          python::arg("useBondOrder") = true, python::arg("atomInvariants") = 0,
          python::arg("fromAtoms") = 0,
-         python::arg("atomBits") = python::object()),
+         python::arg("atomBits") = python::object(),
+         python::arg("bitInfo")=python::object()),
+         docString.c_str(),
+         python::return_value_policy<python::manage_new_object>());
+    python::scope().attr("_RDKFingerprint_version")=RDKit::RDKFingerprintMolVersion;
+
+
+         python::def("UnfoldedRDKFingerprintCountBased", wrapUnfoldedRDKFingerprintMol,
+                     (python::arg("mol"),python::arg("minPath")=1,
+                      python::arg("maxPath")=7,python::arg("useHs")=true,
+                      python::arg("branchedPaths")=true,
+                      python::arg("useBondOrder")=true,
+                      python::arg("atomInvariants")=0,
+                      python::arg("fromAtoms")=0,
+                      python::arg("atomBits")=python::object(),
+                      python::arg("bitInfo")=python::object()),
         docString.c_str(),
         python::return_value_policy<python::manage_new_object>());
     python::scope().attr("_RDKFingerprint_version") =
```

Comment:
Isn't this being added twice now (once here and once above on line 1547)?
If both need to be included (and I don't necessarily think they do, since we don't have versions for every form of every fingerprint) I think you mean one of them to be named '_UnfoldedRDKFingerprint_version`